api/src/main/java/jakarta/enterprise/inject/build/spi/AnnotationBuilder.java

```diff
@@ -1,4 +1,4 @@
-package jakarta.enterprise.inject.build.compatible.spi;
+package jakarta.enterprise.inject.build.spi;
```

Comment:
As mentioned on today's call, after the rename, it sends the impression the package is build SPIs, which are different from the attended purpose (works under both runtime and build time). If lite is not suitable to be used in the package name, how about `core` instead?